torch/_dynamo/eval_frame.py

```diff
@@ -1229,17 +1229,34 @@ def patch():
                 DistributedDataParallel._inside_ddp_forward, recursive=False
             )
 
-        from ..optim import adagrad, adam, adamax, adamw, asgd, nadam, sgd
+        from ..optim import (
+            adadelta,
+            adagrad,
+            adam,
+            adamax,
+            adamw,
+            asgd,
+            nadam,
+            rprop,
+            sgd,
+        )
 
-        for opt_mod in adagrad, adam, adamax, adamw, asgd, nadam, sgd:
-            multi_tensor_fn_name = f"_multi_tensor_{opt_mod.__name__.split('.')[-1]}"
+        for opt_mod in adagrad, adam, adamax, adamw, asgd, nadam, sgd, adadelta, rprop:
+            opt_name = opt_mod.__name__.split(".")[-1]
+            multi_tensor_fn_name = f"_multi_tensor_{opt_name}"
```

Comment:
I'm guessing we will be enabling this one eventually once foreach is happy?

Reply:
Yes